PiwikTracker/URLSessionDispatcher.swift

```diff
@@ -25,6 +25,10 @@ final class URLSessionDispatcher: Dispatcher {
         return currentUserAgent.appending(" PiwikTracker SDK URLSessionDispatcher")
     }()
     
+    func setUserAgent(ua: String) {
+        self.userAgent = ua
+    }
```

Comment:
If you remove the setter function from the protocol, you could also remove this function here.